dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/vitals/VitalFrameCallbackTest.kt

```diff
@@ -157,6 +158,8 @@ internal class VitalFrameCallbackTest {
         val expectedRawFrameRate = ONE_SECOND_NS.toDouble() / frameDurationNs.toDouble()
         val expectedFrameRate = expectedRawFrameRate / deviceRefreshRateScale
 
+        assumeTrue(expectedFrameRate in VitalFrameCallback.VALID_FPS_RANGE)
```

Comment:
Indeed that would be a proper fix. I wanted just to cut the corner, considering that possible ranges already covered by another tests (so was thinking it would be ok to skip this test sometimes).

I pushed the update.